packages/kbn-reporting/export_types_helpers_public/job_types.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const CSV_JOB_TYPE = 'csv_searchsource';
```

Comment:
These seem to be exported from 3 places. Check the code for `export const .*_JOB_TYPE =`